_raml/apps/products/products.raml

```diff
@@ -108,6 +108,18 @@ post:
                 - /productImage⁴<br>
                 - /stocklevel²³<br>
                 - /visible¹<br><br>
+                - /isNew<br>
+                - /newnessDate<br>
+                - /video<br>
+                - /isVATInfoVisible<br>
+                - /isManufacturerPriceRRP<br>
+                - /referencePriceInfo<br>
+                - /isShippingLinkVisible<br>
+                - /customisationInfo<br>
+                - /title<br>
```

Comment:
@bolsu I'm not sure, but since there are several attributes called `title` shouldn't we specify it by writing down the path? Like it is done with e.g. /priceInfo/manufacturerPrice/amount?

Reply:
@dkluever22 not sure what other 'title's you are referring to, there is only one? This is dependent on the code, and must match exactly, path and spelling

Reply:
@bolsu Sorry, my bad. Within the API we have several `title` attributes for different data types, so I thought we maybe needed to add the data type to specify which one is exactly meant. But within product it seems we only have it once, so I guess it doesn't need further clarification.